shared/cfg/machines.cfg

```diff
@@ -63,6 +63,8 @@ variants:
         auto_cpu_model = "no"
         cpu_model = host
         machine_type = s390-ccw-virtio
+        # Usb subsystem not available
+        no usb_d
```

Comment:
I'd suggest adding the restriction into the test suite's configuration file If the variant is only defined for systemtap_tracing.

Reply:
Usually yes, but there is https://github.com/avocado-framework/avocado-vt/pull/1235 which defines the rest of (not only) `usb` filters. With that it'll make sense to have all of them in the same location as we'll easily remove them when/if it's supported. (sorry for the mess, but the review process took so long I had to send PRs that were meant on top of the previous ones and after 3 iterations it started to be a mess).

Reply:
Even though we don't have a policy for what goes into which config file, I agree with @luckyh that the machine files should strive towards keeping configuration that is *machine* related.  Referencing tests from this file doesn't look right.  Also, the common practice is to find the opposite approach (filtering from the test or guest files).

Reply:
Please re-consider this decision, because it's not blocking tests, but rather feature. Here it's not visible enough as it's the remaining left-over, but take a look here: https://github.com/avocado-framework/avocado-vt/pull/1235/commits/88637e65211bf0f367e2068151fede2ad3d283b6

The point is that we expect (at least some) features to be enabled in the future and it'd be hard to look for individual items. It should be way easier to simply remove the line with "usb" related tests once "usb" support is introduced.